scripts/pam-test.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -e
+
+# load common functions
+# shellcheck disable=SC1091
+. ./bash_lib.sh
+
+bb_print_env
+
+control_mariadb_server restart
+
+set +e
+
+res=0
+
+#----------------
+# Basic pam_unix
+#----------------
+
+set -e
+
+sudo tee /etc/pam.d/mariadb <<EOF
+auth required pam_unix.so audit
+account required pam_unix.so audit
+EOF
```

Comment:
could put this as part of the ansible creation rule.

Reply:
https://gitlab.com/mariadb/sysadmin/-/merge_requests/66
@fauust need your help please with a review.

Reply:
Although for me, it's much more clear to keep the VM as clean as possible and let all the related steps required for the PAM auth test be contained in the script itself.

Reply:
I prefer keeping the setup as part of the test. I agree that a clean VM is better.

Reply:
yep, agree, @RazvanLiviuVarzaru https://gitlab.com/mariadb/sysadmin/-/merge_requests/66 looks ok but close it if final decision is to keep this here.